currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -19,9 +19,11 @@ class ResCurrencyRateProviderECB(models.Model):
         selection_add=[('ECB', 'European Central Bank')],
     )
 
-    @api.model
-    def _get_supported_currencies(self, service):
-        if service == 'ECB':
+    @api.multi
+    def _get_supported_currencies(self):
+        self.ensure_one()
+
+        if self.service == 'ECB':
```

Comment:
It's better for indentation to put `if self.service != 'ECB': return super...` and develop the rest of the lines with no indentation.

Reply:
Done